api/models/Campaign.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+/**
+ * Models a DS Campaign.
+ */
+const mongoose = require('mongoose');
+
+const schema = new mongoose.Schema({
+
+  _id: { type: Number, index: true },
+  keyword: String,
+
+  // Properties cached from DS API.
+  title: String,
+  rb_confirmed: String,
```

Comment:
Maybe `rb_confirmation_message`? I just thought why it's not boolean, then realized it's a message.